src/resources/views/event/create.blade.php

```diff
@@ -21,132 +21,88 @@ class="bg-gray-200 hover:bg-gray-300 text-black font-bold py-2 px-4 rounded">Bac
-                            @enderror
+                            @livewire("forms.form", [
+                                'form_elem' => $form_elem,
+                                'type' => "text",
+                                'title' => "Event's name",
+                                'hint' => "The name of the event."])
                         </div>
 
                         <!--  Event Type SECTION  -->
                         <div class="px-4 py-5 bg-white sm:p-6">
                             @php($form_elem = "event_type_id")
                             @php($list = ListEventType::list())
-                            <label for="{{$form_elem}}" class="block font-medium text-md text-gray-700">Event's
-                                Type</label>
-                            @livewire("select-dropdown", ['label' => $form_elem, 'placeholder' => "--
-                            Select the event type --", 'datas' => $list, 'selected_value' =>
-                            old($form_elem, $form_elem)])
-
-                            <small id="{{$form_elem}}Help" class="block font-medium text-sm text-gray-500">The type of
-                                the event</small>
-                            @error($form_elem)
-                            <p class="text-sm text-red-600">{{ $message }}</p>
-                            @enderror
+                            @livewire('forms.form', [
+                                'form_elem' => $form_elem,
+                                'type' => 'select-dropdown',
+                                'title' => 'Event\'s type',
+                                'associated_list' => $list,
+                                'placeHolder' => '-- Select the event type --',
+                            ])
                         </div>
 
 
                         <!--  Country SECTION  -->
                         <div class="px-4 py-5 bg-white sm:p-6">
                             @php($form_elem = "country_id")
                             @php($list = ListCountry::list())
-                            <label for="{{$form_elem}}" class="block font-medium text-md text-gray-700">Event's
-                                Country</label>
-                            @livewire("select-dropdown", ['label' => $form_elem, 'placeholder' => "--
-                            Select the event country --", 'datas' => $list, 'selected_value' =>
-                            old($form_elem, $form_elem)])
-                            @stack('scripts')
-                            <small id="{{$form_elem}}Help" class="block font-medium text-sm text-gray-500">The country
-                                associated to the event</small>
-                            @error($form_elem)
-                            <p class="text-sm text-red-600">{{ $message }}</p>
-                            @enderror
+                            @livewire('forms.form', [
```

Comment:
Does the component allow you to use the search bar ?
If it doesn't add `@stack(scripts)`

Reply:
yup search bar is working